container/openejb-core/src/main/java/org/apache/openejb/core/ivm/naming/IvmContext.java

```diff
@@ -399,7 +408,10 @@ public void destroySubcontext(final Name name) throws NamingException {
     }
 
     public Context createSubcontext(String name) throws NamingException {
-        checkReadOnly();
+        if(checkReadOnly()) {
+            //TODO: throw exception or return null? tomcat returns null
+            return null;
```

Comment:
null is fine if there is a one time - 10 calls will log a single time - log line (warning?)